topcoder-platform/community-app
# Winner tab is not selected when user click on +X users from 'Past cha…

…llenges' #206 

1. Tab selection is now handled by global redux state.
2. For SSR, we set the state for selected tab in the related reducer factory.

To Test Locally - 
1. Goto http://localhost:3000/challenges/30050696 will open the page with details tab, i.e details tab is default selected.
2. Goto http://localhost:3000/challenges/30050696?tab=registrants will open the page with registrants tab selected.
3. Note http://localhost:3000/challenges/30050696?tab=checkpoints causes a problem at the moment, since the Checkpoint related component uses window. As part of another issue #170 - I've removed dependency on window, so it should get solved with that merge.
4. After the page is loaded for the first time, subsequent tab selection is handled on the client via redux.
